hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/records/TokenCreateRecordBuilder.java

```diff
@@ -46,5 +45,5 @@ public interface TokenCreateRecordBuilder extends SingleTransactionRecordBuilder
      * @param tokenAssociation the token association that is created by auto association
      * @return the builder
      */
-    TokenUpdateRecordBuilder addAutomaticTokenAssociation(@NonNull final TokenAssociation tokenAssociation);
```

Comment:
Why was this returning `TokenUpdateRecordBuilder`? Does the new version also need to be returning `TokenUpdateRecordBuilder` instead of `TokenCreateRecordBuilder`?

Reply:
cc @Neeharika-Sompalli @iwsimon

Reply:
I believe it was a copy/paste error.  The method is implemented in two interfaces, and appeared to return the other interface here unintentionally.

Reply:
Yes agree with @jsync-swirlds . Must be a copy/paste error

Reply:
Yes, that was copy/paste error. @jsync-swirlds could you please correct it in your branch?

Reply:
It is corrected.